docs/_source/tutorials/notebooks/training-llm-mistral-sft.ipynb

```diff
@@ -0,0 +1,602 @@
+   "source": [
+    "If you are running Argilla using the Docker quickstart image or Hugging Face Spaces, you need to init the Argilla client with the `URL` and `API_KEY`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Replace api_url with the url to your HF Spaces URL if using Spaces\n",
+    "# Replace api_key if you configured a custom API key\n",
+    "rg.init(\n",
+    "    api_url=\"http://localhost:6900\",\n",
+    "    api_key=\"admin.apikey\",\n",
+    "    workspace=\"admin\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing a FeedbackDataset in Argilla\n",
+    "In Argilla, the [FeedbackDataset](https://docs.argilla.io/en/latest/conceptual_guides/data_model.html#feedback-dataset) is a powerful and widely-configurable class that is in charge of defining the annotation process. In particular, we define [fields](https://docs.argilla.io/en/latest/conceptual_guides/data_model.html#field) and [questions](https://docs.argilla.io/en/latest/conceptual_guides/data_model.html#question).\n",
+    "\n",
+    "The former is in charge of defining the structure for the data that will be annotated, while the latter determines in what way the annotators can annotate the data. In practice, `FeedbackDataset` instances for finetuning LLMs often have \"prompt\" and \"response\" text fields, sometimes alongside a \"context\" text field or some additional metadata (e.g. sample IDs).\n",
+    "\n",
+    "The questions provide a lot of flexibility - text questions allow annotators to provide better responses or suggestions to improve the prompt, while label questions may be used to select if a sample is considered \"good\" or not. With multi-label questions annotators can select whether samples are biased, harmful, incorrect, etc.\n",
+    "\n",
+    "We recommend to set up the questions based on your goals and values for the annotation process. When preparing the training data, all of the annotations will be available, allowing you to carefully curate your training data. For example, based on your annotations you will be able to disregard samples that are (frequently) marked as harmful or low quality, or you can use the annotator-provided suggestions to responses or prompts instead of the existing ones.\n",
+    "Furthermore, in some situations you may only have prompts to begin with, and you can use Argilla to request your annotators to provide responses. You can then use these in your training data.\n",
+    "\n",
+    "### For this tutorial\n",
+    "\n",
+    "For the purposes of this tutorial, we will use a \"prompt\" field with all chat history and the current prompt, as well as a \"response\" field with the response. Additionally, we introduce a \"background\". For the questions we will use one simple `LabelQuestion` that asks the annotator whether the response is `\"Good\"` or `\"Bad\"`."
```

Comment:
background? maybe chat_history is more explicit?

Reply:
This is indeed a bit vague. The "chat history" is all inside of the prompt, the "background" is just some additional bit of text that *some* samples of the OIG dataset have before the whole `<human>:` `<bot>:` steps. The loaded data is a good example, actually.
I elaborate a bit better here: f4b91ad2817d9a77673cac7de680020e5f47ec73